pkg/storage/filepath/jsonfile_rest_test.go

```diff
@@ -315,6 +316,99 @@ func TestFilepathREST_UpdateIdentical(t *testing.T) {
 	require.Equal(t, "2", actual.ResourceVersion)
 }
 
+func TestFilepathREST_ParallelCreateAndWatch(t *testing.T) {
```

Comment:
This test always fails (at least on my machine) when run against code currently in `main` branch